include/bluetooth/gatt.h

```diff
@@ -392,6 +394,15 @@ static inline void bt_gatt_foreach_attr(uint16_t start_handle, uint16_t end_hand
  */
 struct bt_gatt_attr *bt_gatt_attr_next(const struct bt_gatt_attr *attr);
 
+/** @brief Get Attribute handle.
+ *
+ * @param attr Attribute object.
+ *
+ * @return Haandle of the corresponding attribute or zero if the attribute
+ * could not be found.
```

Comment:
Missing a space to align with the @brief

Reply:
Will fix it.